example/.gitignore

```diff
@@ -3,3 +3,4 @@
 .pub/
 build/
 pubspec.lock
+.idea/
```

Comment:
No! These should be in your `~/.gitignore_global` file! 😄 

See https://dart.dev/guides/libraries/private-files

Reply:
It already ignored but I don't know why :D. i will fix now

Reply:
I run ```dartfmt -w .``` on the root project but still a lot of failure, why?